src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,18 @@
+package calculator;
+
+public class StringCalculator {
+
+    public Elements splitWithSpace(String input) {
+        if (input == null || input.isEmpty())
+            throw new IllegalArgumentException("입력 값은 null 이거나 빈 공백 문자가 올 수 없습니다.");
```

Comment:
if 문은 항상 중괄호를 사용하는 것이 컨벤션 입니다!
[7.4 if, if-else, if else-if else 문](https://myeonguni.tistory.com/1596) 를 확인 해주세요! 😄 
```suggestion
        if (input == null || input.isEmpty()) {
            throw new IllegalArgumentException("입력 값은 null 이거나 빈 공백 문자가 올 수 없습니다.");
        }
```

Reply:
헉 구글 문서들에 중괄호를 생략하는 것들이 많고, equals 메서드를 오버로딩하면 
@Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (!(o instanceof Elements)) return false;
        Elements elements1 = (Elements) o;
        return Objects.equals(elements, elements1.elements);
    }
    
 아래와 같이 나오는데, 컨벤션을 따르지 않았던거 였군요. 몰랐네요! 감사합니다!